lib/protocol/routes.js

```diff
@@ -613,6 +613,12 @@ const METHOD_MAP = {
       ]},
     }
   },
+  '/wd/hub/session/:sessionId/appium/compare_images': {
+    POST: {command: 'compareImages', payloadParams: {
+      required: ['mode', 'firstImageData', 'secondImageData'],
```

Comment:
thinking about the param names, why not just `firstImage` and `secondImage`, or `image1` and `image2`? We'll specify the data type as b64-encoded PNG, but the fact that it is data seems superfluous.